### PR TITLE
Montgomp/power bi content

### PR DESCRIPTION
**Pull request by montgomp**
### Testing 🧪
 > n/a

### Checklist ✔️
- [x] Linked associated issue (if present)
- [x] Added relevant labels
- [x] Requested developer reviews
- [x] Request UI review from design / PM
- [x] Verify all code checks are passing

### CODE REVIEW

File: src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx [hunk 1]
```diff
@@ -159,6 +165,28 @@ const PowerBIWidgetBuilder: React.FC<IPowerBIWidgetBuilderProps> = ({
...
+    const renderReportUrlLabel = (props: ITextFieldProps) => {
+        return (
+            <Stack
+                horizontal
+                verticalAlign="center"
+                tokens={{
+                    childrenGap: 4
+                }}
+            >
+                <Label id={props.id} htmlFor={reportUrlId}>
```
Comment (msnyder-msft) [inline src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx:177]: I have never seen `htmlFor` before. Is that like aria-labeled-by but in reverse?

File: src/Resources/Locales/en.json [hunk 2]
```diff
@@ -625,7 +625,7 @@
         "powerBI": {
             "title": "Power BI widget",
             "libraryName": "Power BI Visual",
-            "description": "Render a Power BI visual or tile",
+            "description": "Embed a Power BI visual or tile",
```
Comment (msnyder-msft) [inline src/Resources/Locales/en.json:628]: make sure you bump the loc files by changing the key of this string and push to remote. once all the files have the new key, revert it so they all transalte this update.

File: src/Resources/Locales/en.json [hunk 3]
```diff
@@ -637,15 +637,16 @@
...
                 "label": "Embed URL",
-                "placeholder": "https://embed.powerbi.com/groups/me/reports/{reportId}",
-                "description": "The URL used to embed report artifacts in external tools"
+                "placeholder": "Enter URL",
+                "description": "This URL is used to embed report artifacts in external tools",
+                "tooltip": "This field is required for page and visual name selection"
             },
             "pageName": {
-                "label": "Page Name",
+                "label": "Page name",
```
Comment (msnyder-msft) [inline src/Resources/Locales/en.json:645]: same here. This didnt' propogate to the other files.

File: src/Resources/Locales/en.json [hunk 4]
```diff
@@ -655,7 +656,7 @@
             "errors": {
                 "missingConfiguration": "Widget configuration is missing or is invalid",
                 "missingAdapter": "Please ensure that the Power BI adapter is installed and configured",
-                "missingReportConfiguration": "Please ensure that your widget is fully configured"
+                "missingReportConfiguration": "Configure widget to display details"
```
Comment (msnyder-msft) [inline src/Resources/Locales/en.json:659]: and here.

File: src/Resources/Locales/en.json (hunk 2 above)
Comment (msnyder-msft) [reply src/Resources/Locales/en.json:628]: you can just change teh parent key as well and it will regen all teh children.

Comment (msnyder-msft) [reply src/Resources/Locales/en.json:628]: But the higher you go, the less bonus you get at the end of the year. ;)

File: src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx (hunk 1 above)
Comment (pasanchMSFT) [reply src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx:177]: It gets the label value of the referenced html tag.

**Review by pasanchMSFT — APPROVED** [review]
LGTM

File: src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx (hunk 1 above)
Comment (montgomp) [reply src/Components/PowerBIWidget/Internal/PowerBIWidgetBuilder/PowerBIWidgetBuilder.tsx:177]: Kinda that and also if the user clicks a <label for="myControl" /> the element with id/name "myControl" gets focused.

File: src/Resources/Locales/en.json (hunk 2 above)
Comment (montgomp) [reply src/Resources/Locales/en.json:628]: Straight to the root!!! 🚀

Comment (montgomp) [reply src/Resources/Locales/en.json:628]: Thanks there were some lorem ipsum in there from my initial commit when I just needed *something*.

**Pull request merged**